Makefile

```diff
@@ -78,21 +79,17 @@ assets:
 
 
 # build builds Thanos binary using `promu`.
-.PHONY: build
-build: deps $(PROMU)
+.PHONY: check-git check-bzr build
+build: $(PROMU)
 	@echo ">> building binaries"
 	@$(PROMU) build --prefix $(PREFIX)
 
 # crossbuild builds all binaries for all platforms.
-.PHONY: crossbuild
-crossbuild: deps $(PROMU)
+.PHONY: check-git check-bzr crossbuild
```

Comment:
We do not need these in every `.PHONY` declaration only in 1 place.

Reply:
This is defined at the bottom of the file with the methods

Reply:
Fixed